app/utils/user/index.ts

```diff
@@ -197,7 +197,7 @@ export function isBot(user: UserProfile | UserModel): boolean {
 }
 
 export function isShared(user: UserProfile | UserModel): boolean {
-    return 'remote_id' in user ? Boolean(user.remote_id) : Boolean(user.props?.remote_id);
+    return 'remote_id' in user ? Boolean(user.remote_id) : Boolean((user as UserModel)?.remoteId);
```

Comment:
We shouldn't need the casting, right?

Reply:
I tried it without but TS complained.

Reply:
How is it complaining?

Reply:
![Screenshot 2022-03-28 at 14 44 31](https://user-images.githubusercontent.com/43164508/160381955-687d5f68-ef69-4af3-a5b3-6a4609a83ee7.png)